Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM rust:1.82
+
+RUN cargo install sprocket
```

Comment:
We should probably use `rust:1.82` as a build container but only pull out the `sprocket` binary in the final image.

Reply:
Do you want the binary in a `scratch` image? Or just copy the binary into a clean base image that contains libc?

Reply:
Yeah I think the latter. Just enough to get it running, but cut out all the Rust-related stuff.

Reply:
OK, that's what I have now. Build is in a Rust container, then the binary is copied to a debian container.